src/main/java/com/springweb/dv_spring_web_mongo/controller/ProjectController.java

```diff
@@ -1,5 +1,6 @@
 package com.springweb.dv_spring_web_mongo.controller;
 
+import com.springweb.dv_spring_web_mongo.projectDto.ProjectDTO;
```

Comment:
please rename the package `com.springweb.dv_spring_web_mongo.projectDto` to `com.springweb.dv_spring_web_mongo.dto`